internal/byok/command_list.go

```diff
@@ -69,17 +93,22 @@ func (c *command) list(cmd *cobra.Command, _ []string) error {
 		}
 
 		list.Add(&out{
-			Id:        key.GetId(),
-			Key:       keyString,
-			Cloud:     key.GetProvider(),
-			State:     key.GetState(),
-			CreatedAt: key.Metadata.CreatedAt.String(),
+			Id:                key.GetId(),
+			DisplayName:       key.GetDisplayName(),
+			Key:               keyString,
+			Cloud:             key.GetProvider(),
+			State:             key.GetState(),
+			CreatedAt:         key.Metadata.CreatedAt.String(),
+			ValidationPhase:   key.Validation.GetPhase(),
+			ValidationSince:   key.Validation.GetSince().String(),
+			ValidationRegion:  key.Validation.GetRegion(),
+			ValidationMessage: key.Validation.GetMessage(),
 		})
 	}
 
 	// The API returns a list sorted by creation date already
 	list.Sort(false)
-	list.Filter([]string{"Id", "Key", "Cloud", "State", "CreatedAt"})
+	list.Filter([]string{"Id", "DisplayName", "Key", "Cloud", "State", "ValidationPhase", "CreatedAt"})
```

Comment:
Looks like we're populating all of the validation fields but only displaying one here.